src/components/Filters.js

```diff
@@ -1,39 +1,56 @@
-import { useEffect } from "react";
-import { BsChevronDown } from "react-icons/bs";
 import { MdOutlineKeyboardAlt } from "react-icons/md";
 
 
-const Filters = ({ tag, handleTag, data, tagsArray, filter }) => {
-  useEffect(() => {
-    if (tag !== "") {
-    }
-
-  }, [tag])
+const Filters = ({ tag, handleTag, resetData, tagsArray, filter, handleMidTag, midTag, handleFilterInput, filterInput }) => {
+  // console.log(tag)
+  
   const handleCategoryChange = (e) => {
```

Comment:
You have already done this in the parent component, you can directly use handleTag with onChange